lib/invoca/utils/array.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require_relative './enumerable.rb'
+
+# Invoca ::Array extensions
+class Array
+
+  alias_method :multiply, :*
```

Comment:
Might need to pull in https://github.com/Invoca/hobosupport/blob/master/lib/hobo_support/fixes/module.rb#L7 to make this safe. I will add it if needed after testing these modifications with other gems.

Reply:
Oh, I hope not! I think we can walk away from `alias_method_chain` and its variations. They shouldn't be necessary now that we have `prepend`...which is _so_ much cleaner.
Can you recast this to use `prepend`? Something like:
```
module Invoca
  module Utils
    module ArrayMultiply
      def *(rhs = nil)
        if rhs
          super
        else
          MultiSender.new(self, :map)
        end
      end
    end
  end
end

Array.prepend(Invoca::Utils::ArrayMultiply)
```
(Foreshadowing a comment below... that you can move `MultiSender` out of the `Enumerable` namespace and into ours.)